deploy/docker/clickhouse-setup/docker-compose-local.yaml

```diff
@@ -14,13 +14,15 @@ services:
       - ClickHouseUrl=tcp://clickhouse:9000
       - ALERTMANAGER_API_PREFIX=http://alertmanager:9093/api/
       - SIGNOZ_LOCAL_DB_PATH=/var/lib/signoz/signoz.db
+      - PREFERENCES_PATH=/root/config/preferences.json
```

Comment:
include the same changes for docker swarm and query-service testing-deploy docker-compose files

Reply:
done